### DISCUSSION TITLE
Max users & disable mobile view

### DISCUSSION THREAD

**Discussion by Stoufs** [rejetto/hfs#413] [Ideas]
Is it possible to disable mobile view and is there any way to the max number of anonymous users connected?

**Comment by rejetto** [comment]
it's not clear what you mean by "disable mobile view", tell me more about what you expect.

because of the nature of the web, limiting anonymous users by the nuber is a problem.
A user is not using (necessarily) a single connection, and multiple users can share the same address, so there's no reliable way to distinguish them.
But maybe limiting by the number is not the only solution. If you tell me more about the problem instead of the solution, I may think of something. For example, I could limit the amount of bandwidth for anonymous users.
